src/parsers/pm_io/psse.jl

```diff
@@ -1320,8 +1372,8 @@ function _pti_to_powermodels!(
     _psse2pm_transformer!(pm_data, pti_data, import_all)
     _psse2pm_dcline!(pm_data, pti_data, import_all)
     _psse2pm_facts!(pm_data, pti_data, import_all)
+    _psse2pm_switch_breaker!(pm_data, pti_data, import_all)
```

Comment:
We should move this line upwards before parsing the branches so we can pop out of the dict those branches that aren't lines.